### DISCUSSION TITLE
数据集合作

### DISCUSSION THREAD

**Discussion by UUTTR9** [RapidAI/RapidLatexOCR#18] [Ideas]
你好，我们有mathpix API接口，我们课题组正在构造足够多的手写或印刷体数据（也许在不久的未来超过100万）。也许我们可以合作共同完善Latex ocr模型，如果你对此感兴趣，欢迎联系我们

**Comment by SWHL** [comment]
可以呀，有兴趣参与。咱们邮件联系吧，我的邮箱是liekkaskono@163.com

**Reply by UUTTR9** [reply]
首先感谢您的回复，我们课题组的部分相关工作（构造数据集）即将进入尾声，完成后我将第一时间与您取得联系。
